README.md

```diff
@@ -2,6 +2,27 @@
     <img width="343" src=".github/wagtail.svg#gh-light-mode-only" alt="Wagtail">
     <img width="343" src=".github/wagtail-inverse.svg#gh-dark-mode-only" alt="Wagtail">
     <br>
+    <a href="https://github.com/wagtail/wagtail/actions">
```

Comment:
These should probably be in a separate centre-aligned element rather than inside the `<h1>` - I suspect that screen readers etc will treat them as part of the page title otherwise.

(and now that's got me thinking that the whole concept of badges is inherently breaking accessibility guidelines, since they're communicating information in image-only form that we can't put into the alt text. Hmm...)

Reply:
I moved those badges out of the `<h1>` tag. Good catch on that. Not sure what to do about accessibility though, or if there is even a way to provide dynamic values in the alt tags inside of a README.

Reply:
Thanks! Since the accessibility issue was already present, I think that can be considered out of scope for this PR.